content/guides/models/track/runs/_index.md

```diff
@@ -315,6 +339,32 @@ https://wandb.ai/<team-name>/<project-name>/runs/<run-id>
 
 Where values enclosed in angle brackets (`< >`) are placeholders for the actual values of the team name, project name, and run ID.
 
+### Customize how runs are displayed
+You can customize how runs are displayed in your project from the **Workspace** or **Runs** tab, which share display settings.
+
+To customize which columns are visible:
+1. Above the list of runs, click **Columns**.
+1. Click the name of a hidden column to show it. Click the name of a visible column to hide it.
+  
+  You can optionally search by column name using fuzzy search, an exact match, or regular expressions. Drag columns to change their order.
+1. Click **Done** to close the column browser.
```

Comment:
Do we need a numbered list with only one item?

Reply:
No, but it won't be with the extra spaces in the line before!